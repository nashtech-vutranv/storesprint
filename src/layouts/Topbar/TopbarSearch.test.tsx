import {render} from '@testing-library/react'
import '@testing-library/jest-dom'
import {MemoryRouter} from 'react-router-dom'
import TopbarSearch from './TopbarSearch'

test('renders TopbarSearch', async () => {
  const tree = render(
    <MemoryRouter>
      <TopbarSearch options={[{label: 'label', type: 'type'}]} />
    </MemoryRouter>
  )
  expect(tree).toMatchSnapshot()
})
