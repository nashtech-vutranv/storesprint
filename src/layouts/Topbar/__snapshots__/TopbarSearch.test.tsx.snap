// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TopbarSearch 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="app-search dropdown css-b62m3t-container"
      >
        <span
          class="css-1f43avz-a11yText-A11yText"
          id="react-select-2-live-region"
        />
        <span
          aria-atomic="false"
          aria-live="polite"
          aria-relevant="additions text"
          class="css-1f43avz-a11yText-A11yText"
        />
        <div
          class="react-select__control css-13cymwt-control"
        >
          <span
            class="mdi mdi-magnify search-icon"
          />
          <div
            class="react-select__value-container css-1fdsijx-ValueContainer"
          >
            <div
              class="react-select__placeholder css-1jqq78o-placeholder"
              id="react-select-2-placeholder"
            >
              Search...
            </div>
            <div
              class="react-select__input-container css-qbdosj-Input"
              data-value=""
            >
              <input
                aria-autocomplete="list"
                aria-describedby="react-select-2-placeholder"
                aria-expanded="false"
                aria-haspopup="true"
                autocapitalize="none"
                autocomplete="off"
                autocorrect="off"
                class="react-select__input"
                id="react-select-2-input"
                role="combobox"
                spellcheck="false"
                style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                tabindex="0"
                type="text"
                value=""
              />
            </div>
          </div>
          <div>
            <div
              class="input-group react-select__indicators css-1hb7zxy-IndicatorsContainer"
            >
              <button
                class="btn btn-primary input-group-text"
              >
                Search
              </button>
            </div>
          </div>
        </div>
        <input
          name="search-app"
          type="hidden"
          value=""
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="app-search dropdown css-b62m3t-container"
    >
      <span
        class="css-1f43avz-a11yText-A11yText"
        id="react-select-2-live-region"
      />
      <span
        aria-atomic="false"
        aria-live="polite"
        aria-relevant="additions text"
        class="css-1f43avz-a11yText-A11yText"
      />
      <div
        class="react-select__control css-13cymwt-control"
      >
        <span
          class="mdi mdi-magnify search-icon"
        />
        <div
          class="react-select__value-container css-1fdsijx-ValueContainer"
        >
          <div
            class="react-select__placeholder css-1jqq78o-placeholder"
            id="react-select-2-placeholder"
          >
            Search...
          </div>
          <div
            class="react-select__input-container css-qbdosj-Input"
            data-value=""
          >
            <input
              aria-autocomplete="list"
              aria-describedby="react-select-2-placeholder"
              aria-expanded="false"
              aria-haspopup="true"
              autocapitalize="none"
              autocomplete="off"
              autocorrect="off"
              class="react-select__input"
              id="react-select-2-input"
              role="combobox"
              spellcheck="false"
              style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
              tabindex="0"
              type="text"
              value=""
            />
          </div>
        </div>
        <div>
          <div
            class="input-group react-select__indicators css-1hb7zxy-IndicatorsContainer"
          >
            <button
              class="btn btn-primary input-group-text"
            >
              Search
            </button>
          </div>
        </div>
      </div>
      <input
        name="search-app"
        type="hidden"
        value=""
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
